packages/screens/ReadyLaunchpadApplications/component/ReadyLaunchApplicationTable.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
```

Comment:
Can you find a base component between all `Table` components ? They have almost the same architecture, and i said to myself that it could be nice to have a common base, it's maybe not possible but can you propose something ?